exercises/anagram/.meta/template.j2

```diff
@@ -0,0 +1,20 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+# Python 2/3 compatibility
+if not hasattr(unittest.TestCase, 'assertCountEqual'):
+    unittest.TestCase.assertCountEqual = unittest.TestCase.assertItemsEqual
```

Comment:
The `import unittest` line is inside the header; this section will need to be moved after that.